### ISSUE TITLE
Throwing Missing Plugin Exception

### ISSUE THREAD

**Issue by anugrahkora** [Abdelazeem777/requests_inspector#32]
Version : 3.4.1

causing repetitve crashes -> 

  Fatal Exception: io.flutter.plugins.firebase.crashlytics.FlutterError: MissingPluginException(No implementation found for method setAccelerationSamplingPeriod on channel dev.fluttercommunity.plus/sensors/method)
       at MethodChannel._invokeMethod(platform_channel.dart:308)

**Comment by Abdelazeem777** [comment]
It's related to the `shake` package because it depend on an old version of `sensor_plus` package
So once they update their package dependencies, this issue will be fixed

For more info please check this [issue](https://github.com/deven98/shake/issues/31)

**Comment by Padi142** [comment]
If anyone is looking for a quick workaround, you can add this to your pubspec.yaml:
```yaml
dependency_overrides:
  sensors_plus_platform_interface: '1.1.3'
```
  
  Seems like the shake library is no longer maintained :/

**Comment by Abdelazeem777** [comment]
Fixed on v4.1.1 🚀.
closing this issue for now, but feel free to reopen it again if you are still facing the same behavior.
